packages/babel-core/test/fixtures/transformation/es6.parameters/rest-member-expression-optimisation/expected.js

```diff
@@ -1,9 +1,9 @@
 var t = function () {
-    var x = arguments[0];
-    var y = arguments[1];
+    var x = arguments.length <= 0 || arguments[0] === undefined ? undefined : arguments[0];
```

Comment:
Is the `|| arguments[0] === undefined` necessary for the v8 optimisation? It seems odd that it would